python/ray/tune/tune.py

```diff
@@ -280,6 +281,12 @@ def run(
             ``ray.tune.callback.Callback`` class. If not passed,
             `LoggerCallback` and `SyncerCallback` callbacks are automatically
             added.
+        max_concurrent_trials (int): Maximum number of trials to run
+            concurrently. Must be non-negative. If None or 0, no limit will
+            be applied. This is achieved by wrapping the ``search_alg`` in
+            a :class:`ConcurrencyLimiter`, and thus this argument is ignored
+            if the ``search_alg`` is already a :class:`ConcurrencyLimiter`.
+            Defaults to None.
```

Comment:
We should raise an error if concurrecny limiter is specified also

Reply:
Oh yeah, we do, just gotta update the docstring